Sources/Hedera/Query.swift

```diff
@@ -217,31 +224,39 @@ extension Query: Execute {
         payment.index
     }
 
-    internal func makeRequest(_ transactionId: TransactionId?, _ nodeAccountId: AccountId) throws -> (Proto_Query, ()) {
+    internal func makeRequest(_ client: Client, _ transactionId: TransactionId?, _ nodeAccountId: AccountId) throws -> (
```

Comment:
Shouldn't have the client in here.

Reply:
Went with passing in ledger id and mirror node networks for context instead of `client`